examples/ingest/confluence/ingest.sh

```diff
@@ -0,0 +1,21 @@
+#!/usr/bin/env bash
+
+# Processes all the documents in all spaces within a confluence domain, using the `unstructured` library.
```

Comment:
maybe a comment here that one can target specific spaces by arg?

Reply:
Agreed, added with [169cafd](https://github.com/Unstructured-IO/unstructured/pull/906/commits/169cafdd078ad42f712980e7ed495ec5abb7b268) now 👍